scripts/ci_build_and_test_in_container.sh

```diff
@@ -58,6 +64,8 @@ Usage: $0
       Basename of the json credentials file to connect to the sccache cloud cache.
   --test-code-style
   --test-documentation
+  --xcode
+      Use "Xcode" as build system generator.
```

Comment:
also...why include this?

Reply:
removed

Reply:
It was for genericity purpose. The script can be executed outside the ci by a local `docker run` call so I was guessing that we may want to use one of the generators supported by the project. 
But I have not tested it so ok to remove it and to focus on the ci only.